src/rgw/rgw_sal_motr.cc

```diff
@@ -445,6 +445,11 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
     store->store_access_key(dpp, y, MGWUserKeys);
   }
 
+  if (!info.user_email.empty()) {
+     MotrEmailInfo MGWEmailInfo(info.user_id.id, info.user_email);
```

Comment:
Similarly for tenant:
`info.user_id.id` -> `info.user_id.to_str()`

Reply:
done